### PR TITLE
revision

### PR DESCRIPTION
**Pull request by Timmarh**
Very nice. I added some small suggestions and some spelling things.

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -41,23 +41,23 @@ In this tutorial, you will learn to:
...
 
 Both R and Python can plot geospatial data and have several frameworks and packages for that. R has a built-in `plot()` function that can visualise any type of data quickly, and all packages make use of it. For more advanced custom visualisation, `ggplot2` can be used. In Python, the standard plotting package is `matplotlib`. Spatial packages often implement their own plotting functions for their own objects, therefore putting multiple objects into one plot can often be more challenging than in R.
 
 Ultimately, the choice of language to use is often decided by interoperability needs (e.g. if you work in a company that uses Python, you will be expected to also write in Python, so that your script can be used in a pipeline) and personal preference.
 
 ## Running Python and R from Bash
 
-Both the `R` and the `python` interpreters can be run from Bash. Here is an example you can use to execute a script written in either language:
+Both the `R` and the `python` interpreters can be run from Bash. Here is an example of code that is correct in both python and R. You can use this example to execute a script written in either language:
```
Comment (GreatEmerald) [inline index.Rmd]: `Python` should be uppercase here.

File: index.Rmd [hunk 2]
```diff
@@ -118,7 +118,7 @@ type('6')
 class('6')
 ```
 
-You can combine strings to produce longer strings. This is done with the function `paste()` in R and using the `+` operator in Python (as long as all parts are strings).
+You can combine strings to produce longer strings. This is done with the function `paste()` in R and using the `+` operator in Python (as long as all parts are strings).
```
Comment (GreatEmerald) [inline index.Rmd]: This is not very useful :)

File: index.Rmd [hunk 3]
```diff
@@ -67,7 +67,7 @@ Rscript ./script
 python3 ./script
 ```
 
-This allows you to run script files even without having any graphical user interface installed or running, which is the fastest way to run any script from top to bottom. You can use this method to try out the code examples below, or use your favourite integrated development environment (IDE), such as RStudio or RKWard for R and Spyder or Jupyter for Python.
+This allows you to run script files even without having any graphical user interface installed or running, which is the fastest way to run any script from top to bottom. You can use this method to try out the code examples below, or use your favorite integrated development environment (IDE), such as RStudio or RKWard for R and Spyder or Jupyter for Python.
```
Comment (GreatEmerald) [inline index.Rmd]: No, the original spelling "favourite" is correct, make sure to use the British English spell checker ;)

File: index.Rmd [hunk 4]
```diff
@@ -430,8 +437,9 @@ class building:
     self.coordinates = coordinates
 
   def print(self):
-    print(self.name + " is a building that is located at x: "
-      + str(self.coordinates["x"]) + " y: " + str(self.coordinates["y"]))
+    # Note that we use "string formatting". If a "f" is put before a string quote, 
+    # all text inside curly brackets will be executed as plain python.
```
Comment (GreatEmerald) [inline index.Rmd]: Also uppercase Python here

File: index.Rmd [hunk 5]
```diff
@@ -507,11 +515,11 @@ Gaia.print()
 Aurora.print()
 ```
 
-Note that R also has a more formalised type of classes, called S4 classes, that behave a bit more similar to the Python classes, but S4 classes are generally not recommended to use as they are further away from the R philosophy.
+Note that R also has a more formalized type of classes, called S4 classes, that behave a bit more similar to the Python classes, but S4 classes are generally not recommended to use as they are further away from the R philosophy.
```
Comment (GreatEmerald) [inline index.Rmd]: Same here, "formalised" is correct British English.

**Review by GreatEmerald — CHANGES_REQUESTED** [review]
Thanks! A few minor corrections.

**Comment by Timmarh** [comment]
Requested changes are changed. Merge?

**Review by GreatEmerald — APPROVED** [review]
Thanks!

**Pull request merged**